MAiD - Case Management App/force-app/main/default/objects/Form_1634__c/fields/Disability_support_services_received__c.field-meta.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>Disability_support_services_received__c</fullName>
+    <externalId>false</externalId>
+    <inlineHelpText>If the person received disability support services, what were the services received? (select all that apply)</inlineHelpText>
```

Comment:
@RanadheerRG for future, on general note we should ask Lara to provide us with description for the field.

Reply:
Noted Sunil, I will take a note from next requirements